internal/ansible/runner/eventapi/types.go

```diff
@@ -122,3 +122,15 @@ func (je JobEvent) IgnoreError() bool {
 	}
 	return false
 }
+
+// Rescued - Detects whether or not a task was rescued
+func (je JobEvent) Rescued() bool {
+	if rescued, contains := je.EventData["rescued"]; contains {
+		for _, v := range rescued.(map[string]interface{}) {
+			if int(v.(float64)) == 1 {
```

Comment:
This is just a drive-by comment :) but I happened to look at this PR and wondered -- is it always the case that "rescued" will only ever be 0 or 1? Is it possible multiple tasks could be rescued and thus this count could be 2, or 3, or ???

In which case, wouldn't it be more correct to say:

```
if int(v.(float64)) >= 1 
```